substratevm/src/com.oracle.svm.core.posix.jdk11/src/com/oracle/svm/core/posix/jdk11/linux/Target_jdk_net_LinuxSocketOptions.java

```diff
@@ -0,0 +1,300 @@
+/* This code is free software; you can redistribute it and/or modify it
```

Comment:
Please fix the license header. Is missing the ` Copyright (c) 2019, 2019, Oracle and/or its affiliates. All rights reserved.
 * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.` part.